sources/Outputs/src/Par.cpp

```diff
@@ -194,8 +208,8 @@ Par::writeGenerator(const algo::GeneratorDefinition& def, const std::string& bas
   std::string hashIdStr = std::to_string(hashId);
   auto set = boost::shared_ptr<parameters::ParametersSet>(new parameters::ParametersSet(hashIdStr));
   //  Use the hash id in exported files to prevent use of non-ascii characters
-
-  set->addParameter(helper::buildParameter("generator_KGover", 1.));
+  double value = (DYN::doubleIsZero(def.targetP)) ? 0. : 1.;
```

Comment:
These 2 values "0" and "1" could be factorized

Reply:
How would you do that ?

Reply:
by defining them as constants of the class

Reply:
It is not very clear to me what you mean 
Do you want to add two variables in the class like double zero = 0.; double one = 1.; ?